_posts/02-01-01-Code-Style-Guide.md

```diff
@@ -2,60 +2,43 @@
 anchor: code_style_guide
 ---
 
-# Code Style Guide {#code_style_guide_title}
+# কোড স্টাইল গাইড {#code_style_guide_title}
 
-The PHP community is large and diverse, composed of innumerable libraries, frameworks, and components. It is common for
-PHP developers to choose several of these and combine them into a single project. It is important that PHP code adhere
-(as close as possible) to a common code style to make it easy for developers to mix and match various libraries for
-their projects.
+পিএইচপি একটি বিশাল ও ভিন্ন ধরণের কমিউনিটি এবং এটি অসংখ্য লাইব্রেরি, ফ্রেমওয়ার্ক এবং অন্যান্য কম্পোনেন্টস দ্বারা গঠিত। পিএইচপি ডেভেলপাররা সাধারণত এইগুলোর মধ্যে কয়েকটি নির্বাচন করেন এবং সেগুলো একত্রিত করে একটি প্রজেক্টে ব্যবহার করেন। PHP কোডিং এর ক্ষেত্রে একটি সাধারণ কোড স্টাইল মেনে চলা (যতটুকু সম্ভব) খুবই গুরুত্বপূর্ণ যেন ডেভেলপারেরা তাদের প্রজেক্ট এর জন্য সহজেই বিভিন্ন লাইব্রেরি একত্রিত করে ও সাদৃশ্য বিবেচনা করে কাজ করতে পারেন।
 
-The [Framework Interop Group][fig] has proposed and approved a series of style recommendations. Not all of them related
-to code-style, but those that do are [PSR-0][psr0], [PSR-1][psr1], [PSR-2][psr2] and [PSR-4][psr4]. These
-recommendations are merely a set of rules that many projects like Drupal, Zend, Symfony, Laravel, CakePHP, phpBB, AWS SDK,
-FuelPHP, Lithium, etc are adopting. You can use them for your own projects, or continue to use your own
-personal style.
+[Framework Interop Group][fig] এর দ্বারা প্রস্তাবিত এবং অনুমোদিত কিছু নীতিমালা আছে. যদিও এর সবগুলোই কোড স্টাইল সম্পর্কিত নয়, কিন্তু যেগুলো এ সম্পর্কিত সেগুলো হল [PSR-0][psr0], [PSR-1][psr1], [PSR-2][psr2] এবং [PSR-4][psr4]. এই নীতিমালাগুলো কেবল কিছু নিয়মের সমন্বয় যেগুলো Drupal, Zend, Symfony, Laravel, CakePHP, phpBB, AWS SDK, FuelPHP, Lithium এর মত প্রজেক্টও মেনে চলে। যে কেউ ইচ্ছা করলেই এই নিয়মগুলো মেনে চলতে পারে অথবা তার নিজস্ব স্টাইল মেনে চলতে পারে।
 
-Ideally, you should write PHP code that adheres to a known standard. This could be any combination of PSRs, or one
-of the coding standards made by PEAR or Zend. This means other developers can easily read and work with your code, and
-applications that implement the components can have consistency even when working with lots of third-party code.
+আদর্শভাবে, কোন একটি প্রচলিত নিয়ম অনুসরণ করে পিএইচপি কোড লিখা উচিৎ। তা হতে পারে অনেকগুলো PSR এর একত্রীকরণ অথবা PEAR বা Zend এর তৈরিকৃত কোন মানদণ্ড। অর্থাৎ অন্যান্য ডেভেলপারেরা যেন সহজেই আপনার কোড পড়তে পারে ও এটি নিয়ে কাজ করতে পারে এবং যেসব অ্যাপ্লিকেশন্স কম্পোনেন্টগুলোকে ইমপ্লিমেন্ট করে তাদের মাঝে যেন সামঞ্জস্যতা থাকে এমনকি যখন অসংখ্য থার্ড-পার্টি কোড নিয়ে কাজ করা হয়।
```

Comment:
In my view, "আদর্শভাবে, কোন একটি প্রচলিত নিয়ম অনুসরণ করে পিএইচপি কোড লিখা উচিৎ" simply can be "কোন একটি প্রচলিত নিয়ম অনুসরণ করে পিএইচপি কোড লিখা উচিৎ". Because the word "উচিৎ" already covers the expression of "Ideally".

Another suggestion:
" তা হতে পারে অনেকগুলো PSR এর একত্রীকরণ" => " তা হতে পারে কতগুলো PSR এর সমন্বয়"